src/coreclr/tools/dotnet-pgo/Program.cs

```diff
@@ -767,11 +766,16 @@ static bool IsUnknownTypeHandle(int handle)
                     var best = histogram.OrderByDescending(h => h.Count()).First();
                     Trace.Assert(best.Key.AsType != null);
                     int likelihood = best.Count() * 100 / totalCount;
+                    // The threshold is different for interfaces and classes.
+                    // A flag in the Other field of the TypeHandleHistogram*Count entry indicates which kind of call site this is.
+                    const int INTERFACE_FLAG = 0x40000000;
```

Comment:
Should we add this flag to `PgoFormat.cs` instead?

Reply:
Preferably, yes.